### ISSUE TITLE
Math formula position are detected wrongly

### ISSUE THREAD

**Issue by Colanim** [pdfminer/pdfminer.six#490]
### Description

In my PDF I have some math formula. I encounter no problem when reading the file with `pdfminer`, but the position of the math is wrong.

Because of this, `pdfminer` can't group text into meaningful paragraph.

### Files 

[p1.pdf](https://github.com/pdfminer/pdfminer.six/files/5192985/p1.pdf)

### Code

```python
from pdfminer.high_level import extract_pages

for page in extract_pages('p1.pdf'):
    for elem in page:
        print(elem)
```

### Result

> [...]
<LTTextBoxHorizontal(6) 89.997,606.158,241.137,618.158 'Methods of Calculation\n'>
**<LTTextBoxHorizontal(7) 89.997,574.914,221.639,599.143 'The nominal total shear stress \nflow \n'>**
<LTTextBoxHorizontal(8) 119.551,574.914,288.717,585.414 ' at that point, by the following equation:\n'>
<LTTextBoxHorizontal(9) 231.790,588.643,541.299,599.143 ' in the side shell or longitudinal bulkhead plating is related to the shear\n'>
**<LTTextBoxHorizontal(10) 222.627,565.050,231.783,576.984 'fs\n'>**
<LTTextBoxHorizontal(11) 138.131,550.102,239.360,560.750 'N/t, kN/cm2 (tf/cm2, Ltf/in2)\n'>
<LTTextBoxHorizontal(12) 138.125,530.049,261.020,539.049 'shear flow,  kN/cm (tf/cm, Ltf/in.)\n'>
<LTTextBoxHorizontal(13) 138.125,512.049,254.864,521.049 'thickness of the plating, cm (in.)\n'>
[...]

### Problem

As we can see on this screen shot :

![image](https://user-images.githubusercontent.com/43774355/92562400-e1855900-f2b0-11ea-95cd-12bfbf15dc6a.png)

I am supposed to find a single line containing :

> The nominal total shear stress fs in the side shell or longitudinal bulkhead plating is related to the shear
flow N at that point, by the following equation:

But instead, I have :

* `The nominal total shear stress \nflow \n` located at `y=574` _(Nothing wrong)_
* `fs\n` located at `y=565` _(Wrong position ! Should be on the same line as previous)_
* [...]

Because the math is located at the wrong position by `pdfminer`, paragraph can't be gather to meaningful sentences

---

**Is there anyway to fix this problem ?**

**Comment by jstockwin** [comment]
I agree there is an issue here... the bounding box for the `fs` is way too low, and this is causing the rest of the text to be interpreted wrong.

![image](https://user-images.githubusercontent.com/8227759/92574679-517aeb80-f27f-11ea-95ff-b1d5003fdea7.png)

I'm afraid I am not very familiar with the part of the code than handles this, and don't have much time at the moment, so am not sure what a fix might be at this stage.

**Comment by Colanim** [comment]
Hey thanks for the fast answer ! 

Quick question, how did you display the boxes in blue like this ? In my browser, the bounding boxes are fine.

Is it a debug tool specific to `pdfminer` ?

**Comment by jstockwin** [comment]
Hey! It's actually using a tool I built on top of `pdfminer.six` (py-pdf-parser), so no it's not a debug tool specific to `pdfminer`, but for me it allows me to check these things quite quickly. I have seen other people posting similar screenshots, so I think it may be possible some other way too.

I notice that when I highlight the text in my PDF viewer, certain lines appear way bigger than they should, so it could actually be some problem with the PDF.

In the following screenshot, I am highlighting "shear flow" and it works correctly:
![image](https://user-images.githubusercontent.com/8227759/92578444-c0f2da00-f283-11ea-9819-07c4df1cdb69.png)

However, in this one I am highlighting "The nominal total shear", and it's too big (probably down to where that wrong character box is showing):
![image](https://user-images.githubusercontent.com/8227759/92578522-db2cb800-f283-11ea-9a43-7b78da631536.png)

This does lead me to think that perhaps it's some way that the PDF is generated... however we may somehow be able to fix this I don't know.

**Comment by Colanim** [comment]
Thanks for your answer !

There must be a way, because my browser is fine :

![image](https://user-images.githubusercontent.com/43774355/92665124-b5191d80-f340-11ea-912e-f0559e765d51.png)

I will search more...

**Comment by yeus** [comment]
I wonder if this is the same problem as  #618, with the "descent variable " somehow getting the wrong value...
